---
title: "IMCA-CAT Summer Internship"
date: 2019-11-22T13:49:27-08:00
tags: ['APS', 'crystallography', 'research']
draft: true
---

![](/projects/images/imca.png)

# Background

Over the summer of 2019 I worked at thee Industrial
Macromolecule Crystallization Association - Collaborative
Access Team of Argonne National Laboratory. IMCA-CAT
is a research collaboration funded by a number of
drug companies like Phifzer and Merck who want to have
consistent and predictable access to the Advanced
Photo Source's (APS) synchrotron for X-ray diffraction studies of macromolecules.

{{< figure src="/projects/images/Phillips_sealed_tube.jpg"
title="The APS synchrotron ring at Argonne National Labs">}}

Synchrotrons, like the APS, use powerful magnets to bend
high-speed electrons around their circumference which produces extremely high-intensity X-rays on the tangents of the synchrotron ring. Laboratories along the ring use these high intensity X-rays for many different kinds of interesting experiments, namely X-ray diffraction studies.

X-ray diffraction, in the biological context, is currently the most popular method for determining what a macromolecule (usually a protein) actually looks like at the atomic scale. Having this kind of high-resolution picture of the shape of a protein or other macromolecule
is extremely useful because it provides researchers with important information on how that macromolecule might interact with small molecules like drugs or other proteins. 

Having this kind of insight when attempting to develop a new drug means you can
the scale of your design process can shrink down to the atomic level. 
You can ask and answer questions like, what would a drug look like that could 
bind at this specific location of our target protein that is involved in 
pathway related to tumor growth?

Designing drugs with this paradigm is the bread and butter of researchers 
working with IMCA-CAT. For my project with them, I collaborated with 
researchers from the University of Washington in St. Louis Medical 
School, Dr. Gaya Amarasinghe and Dr. Daisy Leung to learn and apply the
 methodologies of structure based drug design to the Ebola virus.

# VP30: a possible Ebola drug target?

In 2018 Dr.'s Amarasinghe and Leung co-authored a paper
titled *Protein Interaction Mapping Identifies RBBP6 as a Negative Regulator of Ebola Virus Replication* wherein 
they identified the RBBP6 human protein as a negative
regulator of Ebola virus replication. RBBP6 is a host (human) ubiquitin ligase and was first identified as a protein of interest in this study through an analysis of the human-Ebola protein interactome. Normally, VP30 complexes with the viral nucleo-protein or NP
which results in a de-phosphorylation reaction that allows VP30 to participate in RNA transcription. This is a critical interaction since if the Ebola virus can not create RNA it cannot replicate and continue to infect other cells. This makes it an enticing potential target for drug devlopment. 

{{< figure src="/projects/images/ebola_genome.png"
title="Map of the Ebola virus genome showing the six protein coding genes. Batra et al 2018.">}}

Amarasinghe and Leung were subsequently able to show that
Ebola virus replication in human ([HeLa](https://en.wikipedia.org/wiki/HeLa)) cells was significantly reduced in the presence of the RBBP6 protein and a crystal structure of RBBP6 in complex with VP30 revealed a specific binding motif (sequence of amino acids)important to the interaction: **XXXXXPPXPXYX**. 

{{< figure src="/projects/images/vp30_complex.png"
title="Crystal structure of VP30 - RBBP6 complex. Batra et al 2018.">}}

Using this motif Amarasinghe and Leung identified three additional human proteins shown to interact with VP30 and contain this same binding motif which are circled in red in the figure below. 

{{< figure src="/projects/images/vp30_interactome.png"
title="VP30 and human protein interactome. Batra et al 2018.">}}

This is where my role in the project starts to begin. No structural data for the interaction of
these proteins with VP30 existed and crystallization protocols which where used to obtain
crystals of both VP30 and RBBP6 complexes failed when applied with these three novel protein
complexes. My goal was therefore to determine successful co-crystallization conditions for each of
these protein-VP30 complexes and then determine their 3D structure through x-ray diffraction. 

# Protein crystallization

Before you can actually diffract a protein, you need to crystalize it. This can be an extremely difficult and labor intensive endeavour (See Polo for more details on this.) 

To get my foot in the door with protein crystallization
protocols I practiced setting up standard vapor-diffusion
experiments using lysozyme. Lysozyme has a reputation for its willingness to crystalize and so is kind of like the "Hello World" program of crystallography. 

Due to time constraints and the small amount of protein I had available, I decided it would be best
to leverage IMCA-CAT's resources and ship the samples directly to the High-Throughput Crystallography Screening Center (HTCSC) at Hauptmann-Woodward Medical Research Institute, which is capable of screening thousands of different crystallization conditions using a small amount of protein 

{{< figure src="/projects/images/test_pig.png"
title="Images from a complete 1536 well screen at the HTCSC">}}

While the samples were being screened I would get back ~5000 images a week to
review for evidence of a succesfuly crystalization and so I got to work
trying to figure out what would be the optimal way avoid doing this. This
eventually turned into a much larger project which we are expecting to be published
in the [Journal of Applied Crystalography](https://journals.iucr.org/j/) very soon. You can read more about it
in [at this link]({{< ref "Polo" >}})


Eventually, we where able to produce a total of five crystals. Three of one of
protein and one each of the other two. For this I owe a huge thanks to Dr. Sarah
Bowman and the members of the HTCSC. The samples were packed up in liquid nitrogen
and shipped back to Illinois to face the music under the ionizing limelight of the
IMCA beamline. 

# X-ray diffraction

Finally I had made it to the beamline with at least
a few protein crystals. With the help of Dr. Erica Duguid I loaded the crystals up into the robotic handler, and with her looking over my shoulder, sat down at the controls of the APS beamline, ready to blast my crystals with a DNA-shattering dose of X-rays. 

I had five samples to diffract. We loaded the first one
and opened the aperture holding back the syncrotrons
photon fury and rastered it through the beam.

Nothing.

The best (and that word is doing a lot of work)diffraction pattern we got back was the one below. 

![](/projects/images/diff1.png)

This is basically all noise and no signal. And unfortunately the results were basically the same for
the remaining four samples and I walked away with a
nice memento of gettign to operate an APS beamline
but no real data to analyze.

# Lost the battle but not the war

While I did not walk away from my internship at IMCA with a Protein Data
Bank entry in hand as I had originally (and extremely naively) had hoped it was
one of my favorite experiences of my undergrad career. I got to dive into the
world of crystallography, make a ton of great connections in that and surrounding
fields, and got to work everyday in the scientific hub that is the APS. 

A large part of why I made the 80-mile-a-day round-trip commute out to
Lemont Illinois was just to be in that exciting research environment and
realizing that was something I really valued was in-part what pushed me through
the grad school application process. 



